### ISSUE TITLE
2025 Front of House A11y Testing

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5048]
In 2024 our last audit identified work required to become WCAG 2.2 AA compatible. We are now testing to see how successful that has been, and what remains to be done. 

Janeway offeres three themes, OLH, Material and Clean. Of these, Clean was the most accessible, and had the most potential for improvement. The underlying technologies used in OLH and Material worked against us rather than assisted with accessibliity improvements.  We have therefore focused on Clean, while making OLH and Material as accessible as we could.  1.9 introduces an 'accessiblity mode' avaliable across all three themes as a user choice.  This is provided as a short term solution to the accessibility failings of the three themes. Further information about how the accessibility mode works, and how it fits with the themes will be released in 1.9, for the purposes of testing, results are needed for all 4 different sets of templates.
